core/templates/dev/head/profile/profile.html

```diff
@@ -85,7 +85,8 @@ <h2 class="oppia-profile-username-small-screen">
         Interests:
         <span ng-if="subjectInterests.length > 0"><br>
           <!-- The following tag should remain on a single line due to http://bugzilla.mozilla.org/show_bug.cgi?id=922773. -->
-          <span ng-repeat="interest in subjectInterests track by $index"><span class="oppia-profile-subject-interest"><[interest]></span></span>
+          <span ng-repeat="interest in subjectInterests track by $index">
```

Comment:
Ah, ok -- if you've tested, and it works fine, that's great.

I was mainly confused because of [this thread](https://github.com/oppia/oppia/pull/1847#discussion_r63644274) -- do you know why the concern mentioned there no longer applies?